Data Model/mainProperties.js

```diff
@@ -0,0 +1,122 @@
+export const data =
+    [
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'jest to',
```

Comment:
@bernardyn Are you going to use "instance of" here and not "type" as it is in ontohgis?

Reply:
The question is, do we want to be closer to wikidata.org and a general approach or closer to our implementation of the ontology.

Reply:
In my opinion - if it possible - the best is to perform a change `type` in ontology to `instance of` and use instance of in wiki :)

Reply:
Unfortunately, we can't do that. Compatibility with ontohgis should be pursued. Make a property to wikidata as external-link.

Reply:
Ok, I will change `instance of` to `Type` to be consistent with ontology. What you mean in case ot the `external-link`? Do you suggest to use this wiki datatype instead the current `datatype: 'wikibase-item'`?

Reply:
Changed `jest to` to `Type`
And created a new external-id property `ontohgis` to link our data with ontohgis ontology